### PR TITLE
Review teamwork process to avoid conflicts and document in README.rev2

### PR DESCRIPTION
**Pull request by Bektaz**
Some additions to pull request #21

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,21 @@
 This repository was created to post javascript files and open issues related to them, fix bugs in codes, write functions and other activities associated with programming in javascript, all while building skills and confidence with GitHub.
 
-
+<<<<<<< HEAD
```
Comment (patrickwalkowicz) [inline README.md]: This seems to be a leftover after resolving a conflict?

**Comment by patrickwalkowicz** [comment]
Thanks for working on the readme!

How about briefly outlining the two contribution workflows that we've discussed [here](https://github.com/codingforeveryone/js/issues/21)? I think it would be a useful reference guide for people using this repo in the future.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,13 @@
 This repository was created to post javascript files and open issues related to them, fix bugs in codes, write functions and other activities associated with programming in javascript, all while building skills and confidence with GitHub.
 
+Here is how the workflow works in this repository:
 
-Your first step is to "fork" this repository into your own GitHub environment. See the guide below for assistance.
+ - For each new challenge, the author creates a new branch named after the task and creates an issue describing the challenge.
```
Comment (Bektaz) [inline README.md]: I changed in your text the name of the branch from function to task, to generalize it, so just in case the challenge is not a function.

**Comment by Bektaz** [comment]
Hi Patrick,

Have a look please, I have just put your offer with minor changes as an instruction :smile:

**Comment by Bektaz** [comment]
And this task served me like huge experience in github :smile: thanks :+1:

**Comment by patrickwalkowicz** [comment]
@Bektaz, that's much better, thank you!

**Pull request merged**